packages/application-generic/src/factories/sms/handlers/gupshup.handler.ts

```diff
@@ -4,7 +4,7 @@ import { GupshupSmsProvider } from '@novu/gupshup';
 
 export class GupshupSmsHandler extends BaseSmsHandler {
   constructor() {
-    super('Gupshup', ChannelTypeEnum.SMS);
+    super('gupshup', ChannelTypeEnum.SMS);
```

Comment:
We are using lowercase 'gupshup' provider id everywhere else, hence it was not picking up the correct provider.